qemu-test-init

```diff
@@ -148,7 +151,7 @@ if [ -n "$SERVICE" ]; then
   rauc status mark-good
 fi
 
-if ninja test -C $BUILD_DIR -v; then
+if meson test -C $BUILD_DIR -v "$MESON_TEST"; then
```

Comment:
I am not sure if this is intentional, but this significantly changes the output since `-v` in `ninja` means "show all command lines while building" while in for `meson test` it means "Do not redirect stdout and stderr".

As a result, one will always get the full log dumped into the console which is possibly not what is desired or intended?

Reply:
Do you have a suggestion on how to keep the behavior similar?

Reply:
Most similar should be to just omit the `-v` flag.